### ISSUE TITLE
Manage Multiple Databases

### ISSUE THREAD

**Issue by benhaynes** [directus/api#363]
#### :rotating_light: MIGRATED FROM REQUESTS.GETDIRECTUS.COM :rotating_light: 

# Manage Multiple Databases
### :+1: = 53
*Created 1 year ago by @benhaynes*
 

-----
### @benhaynes – :+1: (1 year ago)
This would be a global dropdown to change the DB you're connected to.

@hrwebasst: I'd suggest moving this further up the issue list. I have a use case that is probably pretty common. Installing automatically in my docker instance with my site download for development purposes. I have already easily bypassed the login and can add the schema in by default but it would be nice if Directus could be simply pointed at a database the user had perms for so that I can bake this for all of my users.

tl;dr Directus has own database points to one or multiple sub-databases on install would be awesome.

 

-----
### @nums – (1 year ago)
How can you identify the DB on which the user logs on to the API?

Maybe like that?
curl -d "email=user@directus.local&password=pass123&db=mydb" https://account-instance.directus.io/api/1.1/auth/request-token

 

-----
### @detj – :+1: (7 months ago)
... 
 

-----
### @diegogurpegui – (6 months ago)
Regarding the "How can you identify the DB on which the user logs on to the API" I think it would be better to make it part of the URL/path, instead of a query parameter. And leave those only for filtering and things related to the query you are doing. The URL represents the resource you are accessing, and the database instance is part of that resource in my opinion.

 

-----
### @ezanattatray – :+1: (4 months ago)
amazing

**Comment by rijkvanzanten** [comment]
We have this in the form of Projects!
